Resources/Prototypes/Species/moth.yml

```
@@ -62,16 +62,16 @@
       points: 1
       required: false
     Head:
-      points: 1
+      points: 4 # imp
       required: false
     Chest:
-      points: 1
+      points: 4 # imp
       required: false
     Legs:
-      points: 2
+      points: 4 # imp
       required: false
     Arms:
-      points: 2
+      points: 4 # imp
```

```suggestion
      points: 4 # imp 2<4
```
Note original value for future maintenance work.